packages/adapters/solana/src/client.ts

```diff
@@ -47,6 +47,17 @@ export class SolanaAdapter extends AdapterBlueprint<SolanaProvider> {
     this.wallets = options.wallets
   }
 
+  public override construct(params: AdapterBlueprint.Params): void {
```

Comment:
Why do we need to do it here instead of AppKit client? Can we add PR description "the why" also? 🙏🏽

Reply:
This is a solana specific change. W3M solana users had the connection setup on init and with multichain migration this got changed. See no reason to not init when we can instead of waiting for an actual wallet connection.